src/components/Layout.tsx

```diff
@@ -0,0 +1,13 @@
+import { Outlet } from 'react-router-dom';
+import { NavBar } from './NavBar';
+
+export const Layout = () => (
+  <>
+    <NavBar />
+    <section className="section">
+      <div className="container">
+        <Outlet />
+      </div>
+    </section>
```

Comment:
Wrap content inside components to the container to avoid using react fragments in some cases

```suggestion
    <section className="section">
        <Outlet />
    </section>
```